docs/Doxyfile

```diff
@@ -2512,3 +2512,6 @@ GENERATE_LEGEND        = YES
 # This tag requires that the tag HAVE_DOT is set to YES.
 
 DOT_CLEANUP            = YES
+
+#
+CORRADE_UNUSED         = PREDEFINED
```

Comment:
Ah, no, it's the other way around -- there's a `PREDEFINED` option somewhere above, add `CORRADE_UNUSED=` to it (including the equals at the end).

Reply:
Oh, sorry heh

Reply:
So it should be 
`PREDEFINED         = CORRADE_UNUSED=` 
right?